test/y2partitioner/actions/controllers/encryption_test.rb

```diff
@@ -74,17 +79,34 @@
 
         context "and the device is currently encrypted" do
           let(:encrypted) { true }
+          let(:encryption) { double("Encryption", password: "123456", active?: true) }
+
+          before do
+            allow(encryption).to receive(:exists_in_devicegraph?).and_return in_system
+          end
+
+          context "with a preexisting encryption" do
+            let(:in_system) { true }
 
-          it "returns false" do
-            expect(subject.to_be_encrypted?).to eq(false)
+            it "returns true" do
+              expect(subject.show_dialog?).to eq(true)
+            end
+          end
+
+          context "with a new encryption" do
+            let(:in_system) { true }
```

Comment:
It seems to me that you actually wanted to set `in_system` to `false`, isn't it?